src/ViewRendererBundle/objects/WebModules/MTPkgViewRendererSnippetGalleryCore.class.php

```diff
@@ -45,16 +54,21 @@ public function Init()
      * @param bool                          $bCachingEnabled      - if set to true, you need to define your cache trigger that invalidate the view rendered via mapper. if set to false, you should NOT set any trigger
      * @param IMapperCacheTriggerRestricted $oCacheTriggerManager
      *
-     * @return
+     * @return void
      */
     public function Accept(IMapperVisitorRestricted $oVisitor, $bCachingEnabled, IMapperCacheTriggerRestricted $oCacheTriggerManager)
     {
-        $oSnippetDirectory = new TPkgViewRendererSnippetDirectory();
+        $oSnippetDirectory = $this->getSnippetDirectory();
```

Comment:
I switched to getting the instance from the DI container here as the constructor has a bunch of dependencies that weren't accounted for here. I could have ignored this with comments, but fixing the bug was more straight forward than ignoring it.